app/representers/proxy_representer.rb

```diff
@@ -4,7 +4,7 @@ class ProxyRepresenter < ThreeScale::Representer
   property :service_id
   property :endpoint
   property :deployed_at
-  property :api_backend
+  property :api_backend, render_nil: true
```

Comment:
I don't understand why now we render it even when it is `nil`. Although I don't find it a problem either. But if you consider it somehow important, then it should have a test (and I couldn't find it :thinking: )

Reply:
Well because before the api_backend was never nil, it was always set.
But with Roar, if a value is nil it is not rendered by default which makes the acceptance spec fails